larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -535,6 +559,20 @@ INSTANTIATE_TEST_SUITE_P(
         ::testing::Values(1, 2),  // number of threads
         ::testing::ValuesIn(BConv2DOpTest::GetKernelsTuples(*kKernelMap))),
     TestParam::TestNameSuffix);
+
+TEST(BConv2DTests, BConvErrorTest) {
```

Comment:
I think this test is unnecessary. it is just going to cover whether one if statement inside the code works or not. I would rather replace this one with a new combination of tests which include the activation (see my comment above)

Reply:
I think its important that the op throws an error if you give it wrong parameters. If it doesn't throw an error and the computation is wrong it would be much harder to debug. Especially since this combination "SAME" + fused_relu is something that appears a lot in larq models.
Therefore I think its important to test if this error works as we expect it to. Even though its only one if statement, it check if the error message is being printed and the op doesn't just crash.

> I would rather replace this one with a new combination of tests which include the activation (see my comment above)

What do you mean by this? Test for this error message on all combinations of filter size, strides, dilations, etc?